diff --git a/dist/esm/server/image-optimizer.js b/dist/esm/server/image-optimizer.js
index 4e5a96e6d28492deddccbe360286c07c3035ddd5..fe0bccd10557255071c3ad07653cb46e03bd1335 100644
--- a/dist/esm/server/image-optimizer.js
+++ b/dist/esm/server/image-optimizer.js
@@ -701,7 +701,7 @@ export async function fetchExternalImage(href, dangerouslyAllowLocalIP, count =
         }
     }
     const res = await fetch(href, {
-        signal: AbortSignal.timeout(7000),
+        signal: AbortSignal.timeout(30000),
         redirect: 'manual'
     }).catch((err)=>err);
     if (res instanceof Error) {
diff --git a/dist/server/image-optimizer.js b/dist/server/image-optimizer.js
index 6eba3828d11590c9b1901eae3fd15289d19a492a..2fabb65cc55eb633db550bc572a0e578c24ba05b 100644
--- a/dist/server/image-optimizer.js
+++ b/dist/server/image-optimizer.js
@@ -817,7 +817,7 @@ async function fetchExternalImage(href, dangerouslyAllowLocalIP, count = 3) {
         }
     }
     const res = await fetch(href, {
-        signal: AbortSignal.timeout(7000),
+        signal: AbortSignal.timeout(30000),
         redirect: 'manual'
     }).catch((err)=>err);
     if (res instanceof Error) {
